[Bug] [Dinky-Admin] Some fields of type LocalDateTime lack Json serialization and deserialization annotations

### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar issues.


### What happened

org.dinky.data.model.rbac.Tenant#createTime
org.dinky.data.model.rbac.Tenant#updateTime
These two classes lack the @JsonDeserialize and @JsonSerialize annotations, which results in an error when different versions of jackson-datatype-jsr310 are introduced. 
Error content:
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Java 8 date/time type java.time.LocalDateTime not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" to enable handling (through reference chain: org.dinky.data.result.Result["data"]->org.dinky.data.dto.UserDTO["tenantList"]->java.util.LinkedList[0]->org.dinky.data.model.rbac.Tenant["createTime"])
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77) ~[jackson-databind-2.14.1.jar:2.14.1]

### What you expected to happen

Fix this issue and check if other classes are missing the specified serialization-related annotations

### How to reproduce

Add @JsonDeserialize and @JsonSerialize annotations

### Anything else

_No response_

### Version

1.0.1

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dinky-admin/src/main/java/org/dinky/data/model/LoginLog.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/LoginLog.java
@@ -39,11 +39,12 @@ import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @ApiModel(value = "LoginLog", description = "Login Log Information")
 @TableName("dinky_sys_login_log")
-public class LoginLog implements Serializable {
+public class LoginLog extends DateBaseEntity<LoginLog> implements Serializable {
 
     private static final long serialVersionUID = -3922488670506709018L;
 
@@ -95,22 +96,4 @@ public class LoginLog implements Serializable {
 
     @ApiModelProperty(value = "Access Time", dataType = "String", notes = "Timestamp indicating the time of login")
     private LocalDateTime accessTime;
-
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "String",
-            notes = "Timestamp indicating the creation time of the login log")
-    private LocalDateTime createTime;
-
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "String",
-            notes = "Timestamp indicating the last update time of the login log")
-    private LocalDateTime updateTime;
 }

--- a/dinky-admin/src/main/java/org/dinky/data/model/LoginLog.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/LoginLog.java
@@ -20,26 +20,20 @@
 package org.dinky.data.model;
 
 import org.dinky.mybatis.annotation.Save;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 import java.io.Serializable;
 import java.time.LocalDateTime;
 
 import javax.validation.constraints.NotNull;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
 import com.baomidou.mybatisplus.annotation.IdType;
-import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @ApiModel(value = "LoginLog", description = "Login Log Information")

--- a/dinky-admin/src/main/java/org/dinky/data/model/SysToken.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/SysToken.java
@@ -42,13 +42,14 @@ import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @TableName("dinky_sys_token")
 @ApiModel(value = "SysToken", description = "System Token Information")
-public class SysToken implements Serializable {
+public class SysToken extends DateBaseEntity<SysToken> implements Serializable {
     private static final long serialVersionUID = 3579444102399317143L;
 
     @TableId(value = "id", type = IdType.AUTO)
@@ -91,26 +92,6 @@ public class SysToken implements Serializable {
 
     @ApiModelProperty(value = "Expire End Time", dataType = "Date", notes = "End time for token expiration")
     private Date expireEndTime;
-
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "String",
-            notes = "Timestamp indicating the creation time of the token")
-    private LocalDateTime createTime;
-
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "String",
-            notes = "Timestamp indicating the last update time of the token")
-    private LocalDateTime updateTime;
 
     @ApiModelProperty(
             value = "Creator",

--- a/dinky-admin/src/main/java/org/dinky/data/model/SysToken.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/SysToken.java
@@ -19,22 +19,18 @@
 
 package org.dinky.data.model;
 
+import org.dinky.mybatis.model.DateBaseEntity;
+
 import java.io.Serializable;
 import java.time.LocalDateTime;
 import java.util.Date;
 import java.util.List;
 
 import com.baomidou.mybatisplus.annotation.EnumValue;
-import com.baomidou.mybatisplus.annotation.FieldFill;
 import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
@@ -42,7 +38,6 @@ import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @AllArgsConstructor

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/Menu.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/Menu.java
@@ -20,23 +20,18 @@
 package org.dinky.data.model.rbac;
 
 import org.dinky.mybatis.annotation.Save;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 import java.io.Serializable;
-import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
 import javax.validation.constraints.NotNull;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
 import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
@@ -44,7 +39,6 @@ import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @ApiModel(value = "Menu", description = "Menu Information")

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/Menu.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/Menu.java
@@ -44,6 +44,7 @@ import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @ApiModel(value = "Menu", description = "Menu Information")
@@ -51,7 +52,7 @@ import lombok.NoArgsConstructor;
 @NoArgsConstructor
 @AllArgsConstructor
 @TableName("dinky_sys_menu")
-public class Menu implements Serializable {
+public class Menu extends DateBaseEntity<Menu> implements Serializable {
 
     private static final long serialVersionUID = 8117367692336619625L;
 
@@ -112,24 +113,6 @@ public class Menu implements Serializable {
 
     @ApiModelProperty(value = "Note", dataType = "String", notes = "Additional notes or details about the menu")
     private String note;
-
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "String",
-            notes = "Timestamp indicating the creation time of the menu")
-    private LocalDateTime createTime;
-
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "String",
-            notes = "Timestamp indicating the last update time of the menu")
-    private LocalDateTime updateTime;
 
     @TableField(exist = false)
     @ApiModelProperty(value = "Children", dataType = "List<Menu>", notes = "List of child menus")

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/Role.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/Role.java
@@ -19,25 +19,19 @@
 
 package org.dinky.data.model.rbac;
 
-import java.io.Serializable;
-import java.time.LocalDateTime;
+import org.dinky.mybatis.model.DateBaseEntity;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
+import java.io.Serializable;
+
 import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 /** role */
 @Data
@@ -77,7 +71,6 @@ public class Role extends DateBaseEntity<Role> implements Serializable {
 
     @ApiModelProperty(value = "Note", dataType = "String", notes = "Additional notes or details about the role")
     private String note;
-
 
     @TableField(exist = false)
     @ApiModelProperty(value = "Tenant", dataType = "Tenant", notes = "Associated tenant information")

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/Role.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/Role.java
@@ -37,13 +37,14 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 /** role */
 @Data
 @EqualsAndHashCode(callSuper = false)
 @TableName("dinky_role")
 @ApiModel(value = "Role", description = "Role Information")
-public class Role implements Serializable {
+public class Role extends DateBaseEntity<Role> implements Serializable {
 
     private static final long serialVersionUID = 6877230738922824958L;
 
@@ -77,25 +78,6 @@ public class Role implements Serializable {
     @ApiModelProperty(value = "Note", dataType = "String", notes = "Additional notes or details about the role")
     private String note;
 
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "String",
-            notes = "Timestamp indicating the creation time of the role")
-    private LocalDateTime createTime;
-
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "String",
-            notes = "Timestamp indicating the last update time of the role")
-    private LocalDateTime updateTime;
 
     @TableField(exist = false)
     @ApiModelProperty(value = "Tenant", dataType = "Tenant", notes = "Associated tenant information")

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/RoleMenu.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/RoleMenu.java
@@ -19,31 +19,24 @@
 
 package org.dinky.data.model.rbac;
 
-import java.io.Serializable;
-import java.time.LocalDateTime;
+import org.dinky.mybatis.model.DateBaseEntity;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
+import java.io.Serializable;
+
 import com.baomidou.mybatisplus.annotation.IdType;
-import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @EqualsAndHashCode(callSuper = true)
 @TableName("dinky_sys_role_menu")
 @Data
 @ApiModel(value = "RoleMenu", description = "Role-Menu Relationship Information")
-public class RoleMenu extends DateBaseEntity<RoleMenu>  implements Serializable {
+public class RoleMenu extends DateBaseEntity<RoleMenu> implements Serializable {
 
     private static final long serialVersionUID = -6465347305968663704L;
 
@@ -68,5 +61,4 @@ public class RoleMenu extends DateBaseEntity<RoleMenu>  implements Serializable 
             example = "2001",
             notes = "ID of the menu associated with the role")
     private Integer menuId;
-
 }

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/RowPermissions.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/RowPermissions.java
@@ -19,25 +19,20 @@
 
 package org.dinky.data.model.rbac;
 
+import org.dinky.mybatis.model.DateBaseEntity;
+
 import java.io.Serializable;
-import java.time.LocalDateTime;
 
 import com.baomidou.mybatisplus.annotation.FieldFill;
 import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @EqualsAndHashCode(callSuper = false)

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/RowPermissions.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/RowPermissions.java
@@ -37,12 +37,13 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @EqualsAndHashCode(callSuper = false)
 @TableName("dinky_row_permissions")
 @ApiModel(value = "RowPermissions", description = "Row-Level Permissions Information")
-public class RowPermissions implements Serializable {
+public class RowPermissions extends DateBaseEntity<RowPermissions> implements Serializable {
 
     private static final long serialVersionUID = 8676666963206334660L;
 
@@ -74,26 +75,6 @@ public class RowPermissions implements Serializable {
             example = "user_id = 1001",
             notes = "Expression defining the row-level permissions")
     private String expression;
-
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "String",
-            notes = "Timestamp indicating the creation time of the row-level permissions")
-    private LocalDateTime createTime;
-
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "String",
-            notes = "Timestamp indicating the last update time of the row-level permissions")
-    private LocalDateTime updateTime;
 
     @TableField(exist = false)
     @ApiModelProperty(

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/Tenant.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/Tenant.java
@@ -32,13 +32,14 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 /** tenant use to isolate data */
 @Data
 @EqualsAndHashCode(callSuper = false)
 @TableName("dinky_tenant")
 @ApiModel(value = "Tenant", description = "Tenant model")
-public class Tenant implements Serializable {
+public class Tenant extends DateBaseEntity<Tenant> implements Serializable {
 
     private static final long serialVersionUID = -7782313413034278131L;
     /** id */
@@ -57,22 +58,4 @@ public class Tenant implements Serializable {
     /** is delete */
     @ApiModelProperty(value = "Is Delete", required = true, dataType = "Boolean", example = "false")
     private Boolean isDelete;
-
-    /** create time */
-    @TableField(fill = FieldFill.INSERT)
-    @ApiModelProperty(
-            value = "Create Time",
-            required = true,
-            dataType = "LocalDateTime",
-            example = "2022-02-24 19:58:00")
-    private LocalDateTime createTime;
-
-    /** update time */
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @ApiModelProperty(
-            value = "Update Time",
-            required = true,
-            dataType = "LocalDateTime",
-            example = "2022-02-24 19:58:00")
-    private LocalDateTime updateTime;
 }

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/Tenant.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/Tenant.java
@@ -19,12 +19,11 @@
 
 package org.dinky.data.model.rbac;
 
-import java.io.Serializable;
-import java.time.LocalDateTime;
+import org.dinky.mybatis.model.DateBaseEntity;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
+import java.io.Serializable;
+
 import com.baomidou.mybatisplus.annotation.IdType;
-import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
 
@@ -32,7 +31,6 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 /** tenant use to isolate data */
 @Data

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/User.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/User.java
@@ -42,6 +42,7 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 /**
  * User
@@ -52,7 +53,7 @@ import lombok.EqualsAndHashCode;
 @EqualsAndHashCode(callSuper = false)
 @TableName("dinky_user")
 @ApiModel(value = "User", description = "User Information")
-public class User implements Serializable {
+public class User extends DateBaseEntity<User> implements Serializable {
     private static final long serialVersionUID = -1077801296270024204L;
 
     @ApiModelProperty(value = "ID", dataType = "Integer", notes = "Unique identifier for the user")
@@ -89,23 +90,6 @@ public class User implements Serializable {
     @ApiModelProperty(value = "Is Delete", dataType = "Boolean", notes = "Whether the user is deleted")
     @TableLogic
     private Boolean isDelete;
-
-    @ApiModelProperty(value = "Create Time", dataType = "LocalDateTime", notes = "Timestamp when the user was created")
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    private LocalDateTime createTime;
-
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "LocalDateTime",
-            notes = "Timestamp when the user was last updated")
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    private LocalDateTime updateTime;
 
     @ApiModelProperty(value = "Super Admin Flag", dataType = "Boolean", notes = "Whether the user is a super admin")
     private Boolean superAdminFlag;

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/User.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/User.java
@@ -20,29 +20,22 @@
 package org.dinky.data.model.rbac;
 
 import org.dinky.mybatis.annotation.Save;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 import java.io.Serializable;
-import java.time.LocalDateTime;
 
 import javax.validation.constraints.NotNull;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
 import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableLogic;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 /**
  * User

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserRole.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserRole.java
@@ -37,12 +37,13 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @EqualsAndHashCode(callSuper = false)
 @TableName("dinky_user_role")
 @ApiModel(value = "UserRole", description = "User Role Relationship")
-public class UserRole implements Serializable {
+public class UserRole extends DateBaseEntity<UserRole> implements Serializable {
     private static final long serialVersionUID = -6123386787317880485L;
 
     @ApiModelProperty(value = "ID", dataType = "Integer", notes = "Unique identifier for the user role relationship")
@@ -54,24 +55,4 @@ public class UserRole implements Serializable {
 
     @ApiModelProperty(value = "Role ID", dataType = "Integer", notes = "ID of the role associated with this user")
     private Integer roleId;
-
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "LocalDateTime",
-            notes = "Timestamp when the user role relationship was created")
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    private LocalDateTime createTime;
-
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "LocalDateTime",
-            notes = "Timestamp when the user role relationship was last updated")
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    private LocalDateTime updateTime;
 }

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserRole.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserRole.java
@@ -19,25 +19,18 @@
 
 package org.dinky.data.model.rbac;
 
-import java.io.Serializable;
-import java.time.LocalDateTime;
+import org.dinky.mybatis.model.DateBaseEntity;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
+import java.io.Serializable;
+
 import com.baomidou.mybatisplus.annotation.IdType;
-import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @EqualsAndHashCode(callSuper = false)

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserTenant.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserTenant.java
@@ -38,25 +38,18 @@ package org.dinky.data.model.rbac;
  *
  */
 
-import java.io.Serializable;
-import java.time.LocalDateTime;
+import org.dinky.mybatis.model.DateBaseEntity;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
+import java.io.Serializable;
+
 import com.baomidou.mybatisplus.annotation.IdType;
-import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.annotation.TableName;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
-import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @EqualsAndHashCode(callSuper = false)

--- a/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserTenant.java
+++ b/dinky-admin/src/main/java/org/dinky/data/model/rbac/UserTenant.java
@@ -56,12 +56,13 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dinky.mybatis.model.DateBaseEntity;
 
 @Data
 @EqualsAndHashCode(callSuper = false)
 @TableName("dinky_user_tenant")
 @ApiModel(value = "UserTenant", description = "User Tenant Relationship")
-public class UserTenant implements Serializable {
+public class UserTenant extends DateBaseEntity<UserTenant> implements Serializable {
     private static final long serialVersionUID = -6123386787317880405L;
 
     @ApiModelProperty(value = "ID", dataType = "Integer", notes = "Unique identifier for the user tenant relationship")
@@ -79,24 +80,4 @@ public class UserTenant implements Serializable {
             dataType = "Boolean",
             notes = "Whether the user is a tenant admin for this tenant")
     private Boolean tenantAdminFlag;
-
-    @ApiModelProperty(
-            value = "Create Time",
-            dataType = "LocalDateTime",
-            notes = "Timestamp when the user tenant relationship was created")
-    @TableField(fill = FieldFill.INSERT)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    private LocalDateTime createTime;
-
-    @ApiModelProperty(
-            value = "Update Time",
-            dataType = "LocalDateTime",
-            notes = "Timestamp when the user tenant relationship was last updated")
-    @TableField(fill = FieldFill.INSERT_UPDATE)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
-    private LocalDateTime updateTime;
 }

--- a/dinky-admin/src/main/java/org/dinky/mybatis/model/DateBaseEntity.java
+++ b/dinky-admin/src/main/java/org/dinky/mybatis/model/DateBaseEntity.java
@@ -17,56 +17,56 @@
  *
  */
 
-package org.dinky.data.model.rbac;
-
-import java.io.Serializable;
-import java.time.LocalDateTime;
+package org.dinky.mybatis.model;
 
 import com.baomidou.mybatisplus.annotation.FieldFill;
 import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
 import com.baomidou.mybatisplus.annotation.TableId;
-import com.baomidou.mybatisplus.annotation.TableName;
+import com.baomidou.mybatisplus.extension.activerecord.Model;
 import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
 import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
-
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import org.dinky.mybatis.model.DateBaseEntity;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.dinky.mybatis.annotation.Save;
 
-@EqualsAndHashCode(callSuper = true)
-@TableName("dinky_sys_role_menu")
-@Data
-@ApiModel(value = "RoleMenu", description = "Role-Menu Relationship Information")
-public class RoleMenu extends DateBaseEntity<RoleMenu>  implements Serializable {
+/**
+ * SuperEntity
+ *
+ * @since 2021/5/28
+ */
+@Setter
+@Getter
+@ApiModel(value = "DateBaseEntity", description = "Date Base Entity", parent = Model.class)
+public class DateBaseEntity<T extends Model<?>> extends Model<T> {
 
-    private static final long serialVersionUID = -6465347305968663704L;
-
-    @TableId(type = IdType.AUTO)
+    @TableField(fill = FieldFill.INSERT)
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     @ApiModelProperty(
-            value = "ID",
-            dataType = "Integer",
-            example = "1",
-            notes = "Unique identifier for the role-menu relationship")
-    private Integer id;
+            value = "Create Time",
+            required = true,
+            dataType = "LocalDateTime",
+            example = "2021-05-28 00:00:00")
+    private LocalDateTime createTime;
 
+    @TableField(fill = FieldFill.INSERT_UPDATE)
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     @ApiModelProperty(
-            value = "Role ID",
-            dataType = "Integer",
-            example = "1001",
-            notes = "ID of the role associated with the menu")
-    private Integer roleId;
-
-    @ApiModelProperty(
-            value = "Menu ID",
-            dataType = "Integer",
-            example = "2001",
-            notes = "ID of the menu associated with the role")
-    private Integer menuId;
-
+            value = "Update Time",
+            required = true,
+            dataType = "LocalDateTime",
+            example = "2021-05-28 00:00:00")
+    private LocalDateTime updateTime;
 }

--- a/dinky-admin/src/main/java/org/dinky/mybatis/model/DateBaseEntity.java
+++ b/dinky-admin/src/main/java/org/dinky/mybatis/model/DateBaseEntity.java
@@ -19,24 +19,21 @@
 
 package org.dinky.mybatis.model;
 
+import java.time.LocalDateTime;
+
 import com.baomidou.mybatisplus.annotation.FieldFill;
-import com.baomidou.mybatisplus.annotation.IdType;
 import com.baomidou.mybatisplus.annotation.TableField;
-import com.baomidou.mybatisplus.annotation.TableId;
 import com.baomidou.mybatisplus.extension.activerecord.Model;
 import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
 import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
-import java.io.Serializable;
-import java.time.LocalDateTime;
-import javax.validation.constraints.NotNull;
 import lombok.Getter;
 import lombok.Setter;
-import org.dinky.mybatis.annotation.Save;
 
 /**
  * SuperEntity

